gensim/test/test_atmodel.py

```diff
@@ -110,6 +109,19 @@ def testBasic(self):
         jill_topics = matutils.sparse2full(jill_topics, model.num_topics)
         self.assertTrue(all(jill_topics > 0))
 
+    def testEmptyDocument(self):
+        _local_texts = common_texts + [['only_occurs_once_in_corpus_and_alone_in_doc']]
+        _dictionary = Dictionary(_local_texts)
+        _dictionary.filter_extremes(no_below=2)
+        _corpus = [_dictionary.doc2bow(text) for text in _local_texts]
+        _a2d = author2doc.copy()
+        _a2d['joaquin'] = [len(_local_texts) - 1]
+        try:
```

Comment:
No need try/except section in test, if test raise unexpected exception - this means that test broken